runtime/runtime-params-estimator/src/function_call.rs

```diff
@@ -1,35 +1,38 @@
-use crate::config::GasMetric;
-use crate::gas_cost::GasCost;
-use crate::least_squares::least_squares_method;
+use crate::config::{Config, GasMetric};
+use crate::gas_cost::{GasCost, LeastSquaresTolerance};
 use crate::vm_estimator::create_context;
 use near_primitives::contract::ContractCode;
 use near_primitives::runtime::config_store::RuntimeConfigStore;
-use near_primitives::types::{CompiledContractCache, Gas, ProtocolVersion};
+use near_primitives::types::{CompiledContractCache, ProtocolVersion};
 use near_store::StoreCompiledContractCache;
 use near_vm_logic::mocks::mock_external::MockedExternal;
 use near_vm_runner::internal::VMKind;
-use num_rational::Ratio;
 use std::fmt::Write;
 use std::sync::Arc;
 
-pub(crate) fn test_function_call(metric: GasMetric, vm_kind: VMKind) -> (Ratio<i128>, Ratio<i128>) {
+/// Estimates linear cost curve for a function call execution cost per byte of
+/// total contract code. The contract size is increased by adding more methods
+/// to it. This cost is pure VM cost, without the loading from storage.
+pub(crate) fn function_call_cost_per_code_byte(config: &Config) -> (GasCost, GasCost) {
     let mut xs = vec![];
     let mut ys = vec![];
-    const REPEATS: u64 = 50;
+    let repeats = config.iter_per_block as u64;
+    let warmup_repeats = config.warmup_iters_per_block as u64;
     for method_count in vec![5, 20, 30, 50, 100, 200, 1000] {
```

Comment:
```suggestion
    for method_count in [5, 20, 30, 50, 100, 200, 1000] {
```

still cant' believe this now just works :D